handlers.go

```diff
@@ -207,17 +193,12 @@ type logInfo struct {
 	validationOpts CertValidationOpts
 	// storage stores log data
 	storage Storage
-	// signer signs objects (e.g. STHs, SCTs) for regular logs
-	signer crypto.Signer
+	// signer signs SCTs
+	signSCT signSCT
```

Comment:
Do you want to call this `signer` or `signSCT`?

Reply:
Ah, thanks! Was called signer, and I then renamed it to signSCT.